azurerm_api_management_api - support for `subscription_required` (#4885)
terraform-providers/terraform-provider-azurerm: website/docs/d/api_management_api.html.markdown

* `subscription_key_parameter_names` - A `subscription_key_parameter_names` block as documented below.

* `authentication_settings` - An `authentication_settings` block as documented below.

* `version` - The Version number of this API, if this API is versioned.

* `version_set_id` - The ID of the Version Set which this API is associated with.

---

An `authentication_settings` block exports the following:

> * `subscription_key_required` - Should this API require a subscription key?

Reviewer: ```suggestion
* `subscription_key_required` - Whether the API requires a subscription key or not.
```
